feed/views.py

```diff
@@ -114,6 +111,13 @@ def create(self, request, *args, **kwargs):
         return Response(serializer.data, status=status.HTTP_201_CREATED,
                         headers=headers)
 
+    def perform_create(self, serializer):
+        organization_id = TolaUser.objects. \
+            values_list('organization_id', flat=True). \
+            get(user=self.request.user)
+        rec = serializer.save(organization_id=organization_id)
```

Comment:
¿What means `rec`?

Reply:
It means `record`
I'm gonna change to `obj`, it's better :)